### PR TITLE
Log available networks

### PR DESCRIPTION
**Pull request by FabijanC**
Description Pending

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -60,11 +60,11 @@ extendConfig((config: HardhatConfig, userConfig: Readonly<HardhatUserConfig>) =>
 // add url to alpha network
 extendConfig((config: HardhatConfig) => {
```
Comment (dribeiro-ShardLabs) [inline src/index.ts:61]: Maybe it makes sense to add a default verification URL if the user does not provide one, or at least return an error.

File: test/configuration-tests/with-networks/check.sh [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+echo "Testing no mocha network"
+NETWORK=""    npx hardhat test test/contract-factory-test.ts
+echo "Success"
+
+echo "Testing invalid mocha network"
+NETWORK="$INVALID_NETWORK" npx hardhat test test/contract-factory-test.ts 2>&1 \
+    | tail -n +9 \
+    | head -n 6 \
+    | diff - "$PREFIX/invalid-mocha-network.txt"
+echo "Success"
```
Comment (dribeiro-ShardLabs) [inline test/configuration-tests/with-networks/check.sh:30]: We should also add a test for the verification and invoke tasks

File: src/task-actions.ts [hunk 3]
```diff
@@ -224,21 +207,15 @@ export async function starknetDeployAction(args: any, hre: HardhatRuntimeEnviron
...
-    if (!isTestnet(args.starknetNetwork)) {
-        if (isMainnet(args.starknetNetwork)) {
-            voyagerUrl = VOYAGER_MAINNET_CONTRACT_API_URL;
-        } else {
-            const msg = `Unknown starknet-network provided: ${args.starknetNetwork}`;
-            throw new HardhatPluginError(PLUGIN_NAME, msg);
-        }
+    const network = getNetwork(args.starknetNetwork, hre, "starknet-network");
+    if (!network.verificationUrl) {
+        throw new HardhatPluginError(PLUGIN_NAME, `Network ${args.starknetNetwork} does not support Voyager verification.`);
```
Comment (dribeiro-ShardLabs) [inline src/task-actions.ts:212]: Is there any check for the verification URL? It should be only either testnet or mainnet

**Review by dribeiro-ShardLabs — COMMENTED** [review]
There should be a verification for the verificationURL to be either testnet or mainnet (networks supported by voyager at the moment)

**Review by dribeiro-ShardLabs — APPROVED** [review]

File: test/configuration-tests/with-networks/check.sh (hunk 2 above)
Comment (FabijanC) [reply test/configuration-tests/with-networks/check.sh:30]: The verification tasks are temporarily disabled until (we are blocked by the Voyager team on that), and invoke tasks aren't supported yet, but might be in future.

**Pull request merged**